users/tests.py

```diff
@@ -19,6 +19,16 @@ def _is_authenticated(self):
         user = get_user(self.client)
         return user.is_authenticated
 
+    def test_registration(self):
+        credentials = {
+            'username': 'random@random.com',
+            'password1': self.password,
+            'password2': self.password
+        }
+        response = self.client.post(reverse('registration'), credentials)
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue(self._is_authenticated())
```

Comment:
Need to cover negative cases and check the errors.

Reply:
One more check added